### PR TITLE
Remove collapsible endpoints/events sections (fix #9)

### LINKED ISSUE #9: Don't use HTML <details>/<summary>
Most of the time when I'm browsing documentation, I intend to use ctrl-F to find whatever I'm looking for. Ctrl-F doesn't match anything collapsed in a `<details>`.

(Getting rid of `<details>` would also help with printing, if you wanted to show off the cool spec you put together, for whatever reason.)

### PR DESCRIPTION
**Pull request by bates64**

### CODE REVIEW

File: doc.md [hunk 1]
```diff
@@ -891,46 +884,44 @@ DELETE /api/channels/5678/pins/1234
...
 
 When a message is deleted or edited to remove [the mention of a user](#mentions), all sockets authenticated as the unmentioned user are sent this event. Passed data is in the format `{ messageID }`, where `messageID` is the ID of the message that just stopped mentioning the user.
 
-</details>
-
-<details><summary>Endpoints</summary>
+**Endpoints**
 
 <a name='user-list'></a>
 ### Fetch users [GET /api/users]
```
Comment (towerofnix) [inline doc.md]: I think you meant to make this an H6?

**Review by towerofnix — CHANGES_REQUESTED** [review]
Mostly looks good but a couple things I noticed:

* You haven't *entirely* gotten rid of `<details>` - I still see some of those under the Authentication, Errors, and Permissions sections. These should probably be removed too.

* Why H6's for general headings (e.g. "List roles [GET /api/roles]")? These are extremely small - smaller than the normal text content, actually.

* Why `**`/`<strong>`s for headings like "Endpoints"? `**` isn't actually a heading and wouldn't be taken account in, for example, a table-of-contents generator.

**Comment by bates64** [comment]
> You haven't entirely gotten rid of \<details> - I still see some of those under the Authentication, Errors, and Permissions sections. These should probably be removed too.

This was deliberate - the collapsible sections there are for extra info, which you definitely won't be CTRL+F'ing for.

**Comment by towerofnix** [comment]
'Kay, that's fine I suppose. H6's are still bad though :)

**Comment by bates64** [comment]
@towerofnix resolved.

**Comment by towerofnix** [comment]
Going to merge this with #11.

**Pull request closed without merging**